datamodel/referencedata.d/shipmenteventtypecodes.csv

```diff
@@ -8,4 +8,5 @@ APPR,Approved,Indicates that a document has been approved by the counterpart.
 ISSU,Issued,Indicates that a document has been issued by the carrier.
 SURR,Surrendered,Indicates that a document has been surrendered by the customer to the carrier.
 SUBM,Submitted,Indicates that a document has been submitted by the customer to the carrier
-VOID,Void,Cancellation of an original transport document.
\ No newline at end of file
+VOID,Void,Cancellation of an original transport document.
+CONF,Confirmed,Indicates that the booking is confirmed
```

Comment:
I am thinking that it might be better to use the word "document" instead of "booking" - this would allow CONF to be reused in other scenarios
It would also be good to rename "transport document" to just "document" for same reason
@raisoman what do you think?